source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -3349,7 +3349,7 @@ void Graph::propertyEditor()
                         }
                         else
                         {
-                            nodes->getOutput()->setConnectedNode(_currUiNode->getNode());
+                            uiNode->getOutput()->setConnectedNode(_currUiNode->getNode());
```

Comment:
Code smell: looking at the enclosing if sequence, `_currUiNode->getNode()` will always be null.